common/utils/src/main/resources/error/error-classes.json

```
@@ -433,6 +433,12 @@
     ],
     "sqlState" : "56000"
   },
+  "CLASS_NOT_OVERRIDE_EXPECTED_METHOD" : {
+    "message" : [
+      "<className> must override either <m1> or <m2>."
```

Let's assign proper names to parameters:
```suggestion
      "<className> must override either <method1> or <method2>."
```